ckanext/blob_storage/plugin.py

```diff
@@ -139,10 +143,31 @@ def register_authz_bindings(self, authorizer):
         authorizer.register_authorizer('obj', authz.check_object_permissions,
                                        actions={'update', 'read'},
                                        subscopes=(None, 'data', 'metadata'))
+
+        authorizer.register_authorizer('obj', helpers.check_resource_permissions,
```

Comment:
all of the other authorizer functions are defined in `authz.py`. You have created yours in helpers. The function `check_resource_permissions` should be moved to `authz.py`